src/utils/checkIfParamsMatches.js

```diff
@@ -0,0 +1,11 @@
+const checkIfParamsMatches = (logEntry, match) => {
+  if (logEntry) {
+    if (logEntry.data[0].attributes.childId === match.params.childID) {
```

Comment:
This conditional expression can be simplified.  just return the boolean expression

Reply:
1. Why just taking the first element in the array?
2. BTW, that can also be a potential null pointer exception